src/views/transaction-details/transaction-details.view.tsx

```diff
@@ -0,0 +1,112 @@
+import { FC } from "react";
+import { Link, Navigate, useParams } from "react-router-dom";
+
+import useTransactionDetailsStyles from "src/views/transaction-details/transaction-details.styles";
+import Card from "src/views/shared/card/card.view";
+import Header from "src/views/shared/header/header.view";
+import { ReactComponent as NewWindowIcon } from "src/assets/icons/new-window.svg";
+import { ReactComponent as SpinnerIcon } from "src/assets/icons/spinner.svg";
+import Typography from "src/views/shared/typography/typography.view";
+import { statusText } from "src/views/activity/components/transaction-card/transaction-card.view";
+import { demoData } from "src/views/activity/demo-data";
+import { getTimeFromNow } from "src/utils/time";
+import { convertTokenAmountToFiat } from "src/utils/amounts";
+import TokenIcon from "src/views/shared/token-icon/token-icon";
+import ChainRow from "src/views/transaction-details/components/chain-row/chain-row";
+
+const TransactionDetails: FC = () => {
+  const { transactionId } = useParams();
+  const data = demoData.find((demo) => demo.id.toString() === transactionId);
+  const classes = useTransactionDetailsStyles({ status: data?.status || "initiated" });
+
+  if (!data) {
+    return <Navigate to="/activity" replace />;
+  }
+
+  const { amount, token, timestamp, type, status } = data;
+  return (
+    <>
+      <Header title="Transaction Details" />
+      <Card className={classes.card}>
+        <div className={classes.balance}>
+          <TokenIcon token={token} className={classes.tokenIcon} />
+          <Typography type="h2">{`${amount} ${token.toUpperCase()}`}</Typography>
+          <Typography type="body2">{convertTokenAmountToFiat({ amount, token })}</Typography>
+        </div>
+        <div className={classes.row}>
+          <Typography type="body2" className={classes.alignRow}>
+            Status
+          </Typography>
+          <Typography type="body1" className={classes.alignRow}>
+            <span className={classes.dot} />
+            {statusText[status]}
+          </Typography>
+        </div>
+        <div className={classes.row}>
+          <Typography type="body2" className={classes.alignRow}>
+            {status === "initiated" ? "Confirmation expected in" : "Date"}
+          </Typography>
+          <Typography type="body1" className={classes.alignRow}>
+            {status === "initiated" ? (
+              <>
+                Estimating Time <SpinnerIcon />
+              </>
+            ) : (
+              <>{getTimeFromNow({ timestamp })} ago</>
```

Comment:
I find the time ago very useful, but I think we must provide a way to see the full date.

I'm of the opinion that it is best to put them side by side, but at least, I would show the actual date in a tooltip.

Reply:
We can't decide here, it's a design decission. By the moment I wouldn't worry about this

Reply:
Implemented as requested.